docs/manual/cluster-admin/how-to-use-alert-system.md

```diff
@@ -102,8 +102,8 @@ alert-manager:
 We have provided so far these following actions: 
 
   - `email-admin`: Send emails to the assigned admin.
-  - `email-user`: Send emails to the owners of jobs.
-  - `stop-jobs`: Stop jobs by calling OpenPAI REST API.
+  - `email-user`: Send emails to the owners of jobs. Currently, this action uses the same email template as `email-admin`.
+  - `stop-jobs`: Stop jobs by calling OpenPAI REST API. Be careful about this action because it stops jobs without notifying related users.
```

Comment:
**Be careful about this action because it stops jobs without notifying related users**, please bold this

Reply:
> **Be careful about this action because it stops jobs without notifying related users**, please bold this

Fixed, thanks.